L1Trigger/L1TMuonEndCap/src/PrimitiveConversion.cc

```diff
@@ -679,48 +728,55 @@ void PrimitiveConversion::convert_gem(
     csc_nID += 1;
 
     if (tp_station == 1) {  // neighbor ME1
-      if (not(tp_subsector == 2))
-	{ edm::LogError("L1T") << "tp_subsector = " << tp_subsector; return; }
+      assert(tp_subsector == 2);
```

Comment:
No asserts.

Reply:
This belongs to coding errors, not input/runtime dependent. I think assert should be allowed.